src/I_Ecobee1.xml

```diff
@@ -954,7 +954,7 @@ decompress_lzo_file() {
       if isEmsThermostat(lul_device) then
         return away and setOccupied(session, selection, lul_device, false) or resumeProgram(session, selection, 3)
       else
-        return setClimateHold(session, selection, lul_device, away and "away" or "home")
+        return setClimateHold(session, selection, lul_device, away and "away")
```

Comment:
I don't think `away and "away"` is enough, since some value has to be passed to `setClimateHold` in the 4th argument.  In the function `setAway`, the `away` argument is treated as a boolean, so this function is expected to set either the `home` or `away` climate hold.  I think the ` or "home"` was correct.